documentation/user_docs/docs/manual/Symmetrising_etc.rst

```diff
@@ -761,9 +761,20 @@ dataset (thereby increasing the counts four-fold).
    calculate magnetic form factor on or apply magnetic form factor to the
    dataset provided.
 
+.. Re# 1447 should we reimplement `combine_equivalent_zones` ? See chunk symmeterize_equivalent_zones_description.bak file in this folder when ticket resolved.
+
+
 Limitations
 ===========
 * At present ``symmetrise_sqw``, ``combine_sqw``, and ``rebin_sqw`` work ONLY
   for sqw objects, since they require access to individual detector pixel
   information. The functions will work for any dimensionality of object,
-  however.
\ No newline at end of file
+  however.
+
+.. | Re #1447 to enable  * ``combine_equivalent_zones`` has to perform some memory and hdd-access
+  intensive calculations, which should ideally be performed on `high performance
+  computing cluster
+  <http://www.isis.stfc.ac.uk/groups/excitations/data-analysis-computers/connecting-to-isiscomputendrlacuk-using-nomachine15120.html>`__. The
+  amount of memory used by the code is controlled by ``hor_config`` parameter
+  ``mem_chunk_size`` and is approximately 10 times larger then the amount,
+  specified by this parameter. | replace ::
```

Comment:
```suggestion
..
   Re #1447 to enable * ``combine_equivalent_zones`` has to perform some memory and
   hdd-access intensive calculations, which should ideally be performed on `high
   performance computing cluster
   <http://www.isis.stfc.ac.uk/groups/excitations/data-analysis-computers/connecting-to-isiscomputendrlacuk-using-nomachine15120.html>`__. The
   amount of memory used by the code is controlled by ``hor_config`` parameter
   ``mem_chunk_size`` and is approximately 10 times larger then the amount,
   specified by this parameter.
```

Reply:
and this change does not work either. I had it initially but it work for first line so I need to use `replace` syntaxis

Reply:
Well, it works on my machine. You should not use the replace syntax because that is not what you're doing

Reply:
I suspect the reason it didn't work is that you lacked the newline after the `..`, c.f. [here](https://stackoverflow.com/questions/4783814/how-to-comment-a-string-in-restructured-text)

Reply:
Provided reference suggests putting `..`  for each commented line. It is not what anybody wants.

Reply:
replacement is exactly what I am doing ( where  possible).

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#comments